pkg/policy/snapshotTesting.go

```diff
@@ -0,0 +1,147 @@
+package policy
```

Comment:
Why not `snapshot_testing.go` as filename? I would also add a `snapshot_testing_test.go` for good measure and giggles.

Reply:
good suggestion